src/main/java/io/cdap/plugin/gcp/common/GCPConfig.java

```diff
@@ -145,32 +145,6 @@ public boolean autoServiceAccountUnavailable() {
     return false;
   }
 
-  /**
-   * This method validates the cmek key formatted string.
-   *
-   * @param cmekKey cmek key raw string
-   * @param arguments runtime arguments
-   * @param collector  failure collector
-   * @return parsed CryptoKeyName object if the formatted string is valid otherwise null.
-   */
-  @Nullable
-  public CryptoKeyName getCmekKey(@Nullable String cmekKey, @Nullable Arguments arguments, FailureCollector collector) {
```

Comment:
it's better to keep this here and have it call the utils method. Except the cmekKey value should come from the plugin properties like before instead of passed in (i.e. `getCmekKey(@Nullable Arguments arguments, FailureCollector collector)`). This way the raw unvalidated cmek key is not exposed.